src/Tools/AI Test Toolkit/src/API/AITLogEntryAPI.Page.al

```diff
@@ -0,0 +1,127 @@
+    Extensible = false;
+    DelayedInsert = true;
+
+    layout
+    {
+        area(Content)
+        {
+            repeater(Group)
+            {
+                field(id; Rec.SystemId)
+                {
+                    Caption = 'Id';
+                    Editable = false;
+                }
+                field("aitCode"; Rec."Test Suite Code")
+                {
+                    Caption = 'AIT Code';
+                    Editable = false;
+                    NotBlank = true;
+                    TableRelation = "AIT Test Suite";
+                }
+                field("lineNumber"; Rec."Test Method Line No.")
+                {
+                    Caption = 'Line No.';
+                }
+                field("tag"; Rec.Tag)
+                {
+                    Caption = 'Tag';
+                }
+                field("version"; Rec.Version)
+                {
+                    Caption = 'Version No.';
+                }
+                field("entryNumber"; Rec."Entry No.")
+                {
+                    Caption = 'Entry No.';
+                }
+                field("startTime"; Rec."Start Time")
+                {
+                    Caption = 'Start Time';
+                }
+                field("endTime"; Rec."End Time")
+                {
+                    Caption = 'End Time';
+                }
+                field("codeunitID"; Rec."Codeunit ID")
+                {
+                    Caption = 'Codeunit ID';
+                }
+                field("codeunitName"; Rec."Codeunit Name")
+                {
+                    Caption = 'Codeunit Name';
+                }
+                field("procedureName"; Rec."Procedure Name")
+                {
+                    Caption = 'Function Name';
+                }
+                field("operation"; Rec.Operation)
+                {
+                    Visible = false;
+                    Enabled = false;
+                    Caption = 'Operation';
+                }
+                field("message"; MessageTxt)
+                {
+                    Caption = 'Message';
+                }
+                field("durationMin"; Rec."Duration (ms)")
+                {
+                    Caption = 'Duration (ms)';
+                }
+                field("status"; Rec.Status)
+                {
+                    Caption = 'Status';
+                }
+                field(dataset; Rec."Test Input Group Code")
+                {
+                    Caption = 'Dataset';
+                }
+                field("datasetLineNumber"; Rec."Test Input Code")
+                {
+                    Caption = 'Dataset Line No.';
+                }
+                field("inputData"; this.InputText)
+                {
+                    Caption = 'Input Data';
+                }
+                field("outputData"; this.OutputText)
+                {
+                    Caption = 'Output Data';
+                }
```

Comment:
Consider adding `field(lastModifiedDateTime; Rec.SystemModifiedAt) { }` to properly support webhooks